dubbo-common/src/test/java/org/apache/dubbo/common/threadpool/support/AbortPolicyWithReportTest.java

```diff
@@ -44,4 +50,27 @@ public void run() {
         Thread.sleep(1000);
 
     }
-}
\ No newline at end of file
+
+
+    @Test
+    public void testThreadPoolShutDown() throws InterruptedException {
```

Comment:
I think this UT is meaningless, you should test the method you modified.

Reply:
This unit test answers another member question nicknamed carryxyh。you can see it in [#3207](https://github.com/apache/incubator-dubbo/pull/3207)